### DISCUSSION TITLE
Automatically trust server certificates

### DISCUSSION THREAD

**Discussion by michelelacorte** [FreeOpcUa/opcua-asyncio#710] [Q&A]
Hello everyone, my problem is to automatically verify the certificates sent to me by the server the first time I connect to OPC-UA.
Is there any way?

Trying a windows client this flow works perfectly, so I was wondering how to replicate it in python.

**Comment by oroulet** [comment]
I think there is no code for that. but MR is welcome

**Reply by michelelacorte** [reply]
not even using some bash script?
